cmd/capibmadm/cmd/version/version.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
-	"gopkg.in/yaml.v2"
+	"sigs.k8s.io/yaml"
```

Comment:
please move `sigs.k8s.io/yaml` after line number 28 in the section of `sigs.k8s.io/` imports

Reply:
Done. Could you please share how you figured out the change based on the error message. I tried running `make lint` and `make lint-fix` but it didn't help much.

Reply:
Here is the failure message:
```
cmd/capibmadm/cmd/version/version.go:27: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/IBM) -s prefix(k8s.io) -s prefix(sigs.k8s.io) -s prefix(sigs.k8s.io/cluster-api) -s prefix(sigs.k8s.io/cluster-api-provider-ibmcloud) -s blank -s dot (gci)
	"sigs.k8s.io/cluster-api-provider-ibmcloud/version"
cmd/capibmadm/cmd/version/version.go:28: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/IBM) -s prefix(k8s.io) -s prefix(sigs.k8s.io) -s prefix(sigs.k8s.io/cluster-api) -s prefix(sigs.k8s.io/cluster-api-provider-ibmcloud) -s blank -s dot (gci)
	"sigs.k8s.io/yaml"
```

one of the lint failing wrt to [gci](https://golangci-lint.run/usage/linters/#gci), and the rule set for this project is https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/61e7587fadf00e0ece903645732e9c2b55323a41/.golangci.yml#L59C1-L69C12

So the import order should in this case is:

```golang
import (
	"encoding/json"
	"fmt"
	"io"

	"github.com/pkg/errors"
	"github.com/spf13/cobra"
	"gopkg.in/yaml.v2"

	"sigs.k8s.io/yaml"

	"sigs.k8s.io/cluster-api-provider-ibmcloud/version"
)
```

Reply:
Thank you so much for the explanation @mkumatag! I have made the changes and `make lint ` is passing now.